cmd/retrieve.go

```diff
@@ -0,0 +1,180 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	"archive/zip"
+	"io/ioutil"
```

Comment:
Small bit: sorry I wasn't specific: all stdlib imports are typically grouped together. I _think_ `go fmt ./...` will do this for you